### DISCUSSION TITLE
Extend rules sources with recommended rule emoji

### DISCUSSION THREAD

**Discussion by bc-m** [biomejs/biome#1598] [Ideas]
When reading the rules, I noticed that I now have to switch between "Rules" and "Rules sources" to check whether the rule has already been enabled by recommended rules.

It would be great if the "recommended rule"-emoji could also be seen in the rules sources list.

**Comment by ematipico** [comment]
Do you want to help with this? :)

**Reply by bc-m** [reply]
Yep. What do you think about adding the emojis at both sides? It would help understanding rules are recommend only from source, only from biome or recommended from both?

**Reply by bc-m** [reply]
Oh, currently found this discussion: https://github.com/biomejs/biome/discussions/3 Exactly what I had in mind! 😃

**Comment by Conaclos** [comment]
Thanks for the idea! It is now as you suggested.
